app.js

```diff
@@ -20,9 +27,11 @@ let error = null;
 /* Events */
 
 window.addEventListener('load', async () => {
-    // > Part B: Add a click event listener for the todoEl
-    //      - call the async supabase function to delete all todos
-    //        and get the response
+    const response = await getTodos();
+    error = response.error;
+    todos = response.data;
+    // > Part B: Get todos
+    //      - call the async supabase function to get todos
     //      - set the todos and error state from the response
     //      - if there's an error call displayError
     //      - otherwise, display the todos
```

Comment:
you load the todos and store them in state on window load, but you're not displaying them until you run your function during adding a todo